content/en/docs/distributions/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -107,7 +110,7 @@ export KF_DIR=${BASE_DIR}/${KF_NAME}
 
 # Set the configuration file to use, such as:
 export CONFIG_FILE=kfctl_ibm.yaml
-export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/v1.2-branch/kfdef/kfctl_ibm.v1.2.0.yaml"
+export CONFIG_URI="https://github.com/IBM/manifests/blob/v1.3/distributions/kfdef/kfctl_ibm.v1.3.0.yaml"
```

Comment:
it will respond HTML content if using this URL. please use this URL:

https://raw.githubusercontent.com/IBM/manifests/v1.3/distributions/kfdef/kfctl_ibm.v1.3.0.yaml